tests/integration-tests/tests/performance_tests/test_starccm/test_starccm/pcluster.config.yaml

```diff
@@ -0,0 +1,47 @@
+Region: eu-west-1
```

Comment:
in python code you're injecting `number_of_nodes` but you do not use it in the config file. It is expected to be injected.